src/main/kotlin/no/nav/emottak/eventmanager/kafka/EventReceiver.kt

```diff
@@ -29,7 +29,7 @@ suspend fun startEventReceiver(topic: String, eventService: EventService) {
     KafkaReceiver(receiverSettings)
         .receive(topic)
         .map { record ->
-            log.debug("Processing record: $record")
+            log.debug("Processing record: {}", record)
```

Comment:
Why? Ser at flere bruker $var i stedet for {} som er mindre leslig

Reply:
Fordi det er unødvendige å evaluere logg-blokka hvis debug-logging ikke er enabla. Den blir _alltid_ kjørt om vi lar koden stå slik den er og det vil vi jo ikke

Reply:
I tidssensitive eller ytelseskritiske sammenhgner hadde jeg vært enig, men i dette tilfellet er det neppe noe reell gevinst, og da foretrekker jeg lesbarhet fremfor ytelse.